src/PythonExtensionGen.cpp

```diff
@@ -259,10 +270,10 @@ static __attribute__((unused)) int _convert_py_buffer_to_halide(
 static_assert(PY_MAJOR_VERSION >= 3, "Python bindings for Halide require Python 3+");
 static struct PyModuleDef _moduledef = {
     PyModuleDef_HEAD_INIT,
-    .m_name=MODULE_NAME,
-    .m_doc=NULL,
-    .m_size=-1,
-    .m_methods=_methods,
+    MODULE_NAME,
+    NULL,
+    -1,
+    _methods,
```

Comment:
Why was this deleted?

Reply:
`.<member name>=<value>` syntax isn't supported by MSVC as it is not part of the ISO C++ until C++20.

see https://en.cppreference.com/w/cpp/language/aggregate_initialization

Reply:
Ah, that's good to know. Thanks for the fix, then!